src/pages/bubblegum-v2/burn-cnfts.md

```diff
@@ -0,0 +1,47 @@
+---
+title: Burning Compressed NFTs
+metaTitle: Burning Compressed NFTs | Bubblegum V2
+description: Learn how to burn compressed NFTs on Bubblegum.
+---
+
+The **burnV2** instruction can be used to burn a Compressed NFT and, therefore, remove it from the Bubblegum Tree permanently. To authorize this operation, either the current owner or the delegate authority — if any — must sign the transaction. The instruction accepts the following parameter:
+
+- **Leaf Owner** and **Leaf Delegate**: The current owner of the Compressed NFT and its delegate authority, if any. One of these must sign the transaction.
```

Comment:
Could also be `permanentBurnDelegate` on the collection, in which case of course you will likely want to explain passing in the collection as well